resources/bundles/org.eclipse.core.resources.win32.x86_64/META-INF/MANIFEST.MF

```diff
@@ -1,9 +0,0 @@
-Manifest-Version: 1.0
-Bundle-ManifestVersion: 2
-Bundle-Name: %win32FragmentName
-Bundle-SymbolicName: org.eclipse.core.resources.win32.x86_64;singleton:=true
```

Comment:
This fragment would have to be removed from the eclipse.platform-feature before:
https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/blob/58b6a93fa511d27c86fe255449c86780c03f2940/eclipse.platform.releng/features/org.eclipse.platform-feature/feature.xml#L230-L234

Reply:
Done with https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/pull/2127.